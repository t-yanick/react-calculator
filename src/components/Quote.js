import React from 'react';

function Quote() {
  return (
    <div>
      <h1>Quote of the day</h1>
      <p>
        “A life spent making mistakes is not only more honorable,
        but more useful than a life spent doing nothing.”
      </p>
      {' '}
      <span>Author: Unknown</span>
    </div>
  );
}

export default Quote;
