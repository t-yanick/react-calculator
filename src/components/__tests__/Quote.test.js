import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../Quote';

test('Component renders correctly', () => {
  const component = renderer.create(
    <Quote />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});