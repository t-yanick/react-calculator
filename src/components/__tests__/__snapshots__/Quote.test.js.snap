// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders correctly 1`] = `
<div>
  <h1
    className="quote-title"
  >
    Quote of the day
  </h1>
  <p
    className="quote"
  >
    “A life spent making mistakes is not only more honorable, but more useful than a life spent doing nothing.”
  </p>
   
  <span
    className="quote"
  >
    Author: Unknown
  </span>
</div>
`;
